inbc-program/inbc/parser.py

```diff
@@ -102,6 +102,10 @@ def parse_sota_args(self) -> None:
                                  help='Release date of the applying package - format YYYY-MM-DD')
         parser_sota.add_argument('--username', '-un', required=False, help='Username on the remote server',
                                  type=lambda x: validate_string_less_than_n_characters(x, 'Username', 50))
+        parser_sota.add_argument('--command', '-c', default='update',
```

Comment:
Since the command is always set to update.  I question if we actually need to add that to INBC.  We can just hardcode that to update and if there ever is another option, then we can add it to INBC.

Reply:
@nmgaston Yes agree, but this is just a placeholder, Would you want me to remove this?

Reply:
I would.  I think it's confusing since there isn't any real option at this time.  If I saw that in someone else's software I would really question it.